confgenerator/filter/internal/filter.bnf

```diff
@@ -0,0 +1,246 @@
+dot : '.' ;
```

Comment:
Comment with a link to the public BNF and Cloud Logging docs and a note that the Logging syntax is more complicated than the public BNF?

Reply:
By public BNF are you referring to AIP-160 (https://google.aip.dev/160)?

Reply:
Yeah, and the text file it links to.